backtrace-library/src/main/java/backtraceio/library/common/BacktraceSerializeHelper.java

```diff
@@ -33,6 +33,7 @@ public static String toJson(Object object) {
     }
 
     public static <T> T fromJson(String json, Class<T> type) {
-        return new Gson().fromJson(json, type);
+        Gson gson = new GsonBuilder().setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_DASHES).create();
```

Comment:
maybe it is worth  to extract this line to private method in this file and use it in both places to make sure that we will not have any issues like this in the future

Reply:
Done, thank you for the excellent advice!